packages/@contentful--app-scripts/lib/activate/activate-bundle.js

```diff
@@ -6,8 +6,12 @@ const { createClient } = require('contentful-management');
 async function activateBundle({ accessToken, organization, definition, bundleId }) {
   const activationSpinner = ora('Activating your bundle').start();
   const client = createClient({ accessToken });
-  const currentOrganization = await client.getOrganization(organization.value);
-  const currentDefinition = await currentOrganization.getAppDefinition(definition.value);
+
+  const definitionUrl = `/organizations/${organization.value}/app_definitions/${definition.value}`;
+
+  const currentDefinition = await client.rawRequest({
+    url: definitionUrl,
+  });
```

Comment:
Not quite what I meant.

Look here for the CMA plain client: https://github.com/contentful/contentful-management.js#alternative-plain-api
You don't need to specify the url yourself.

Reply:
Ahh! wasn't aware of that 👍